### ISSUE TITLE
Give applicant homepage a unique title (not default "Civiform")

### ISSUE THREAD

**Issue by neel-a-mehta** [seattle-uat/civiform#1465]
The applicant homepage (the one with the list of programs) has a generic `<title>`: "CiviForm". This is hard for users to understand, and doesn't give much information to people using screen readers.

Instead, we should give that page a more descriptive title, like "Get Seattle Benefits: CiviForm".

## Original tester's report

> ACTUAL RESULT: Page title is not announcing to the user on 'Get benefits' page. Screen reader is announcing 'CiviForm' page title for 'Get benefits' page.
> EXPECTED RESULT: Page title should be unique for each page and it should describe the page's topic, i.e. 'Get benefits - civiForm' instead of 'Civiform'.

**Comment by neel-a-mehta** [comment]
It looks like the homepage title is now `Get benefits - CiviForm — CiviForm`. Can we remove the duplicative `CiviForm`, making it, eg, `Get Benefits — CiviForm`? Or even better, `Get Seattle Benefits — CiviForm`?
